src/pgrepr/src/value.rs

```diff
@@ -267,3 +326,11 @@ pub fn values_from_row(row: Row, typ: &RelationType) -> Vec<Option<Value>> {
         .map(|(col, typ)| Value::from_datum(col, typ))
         .collect()
 }
+
+impl std::fmt::Display for Value {
+    fn fmt(&self, f: &mut std::fmt::Formatter) -> std::fmt::Result {
+        let mut buf = BytesMut::new();
+        self.encode_text(&mut buf);
+        f.write_str(str::from_utf8(&buf).unwrap())
+    }
+}
```

Comment:
I'd kind of prefer that `Value` not have a `Display` impl, because it's not meant to be displayed. If you move the text encoding into `strconv` then you can just use `strconv::format_list` in the SLT runner, like we do for a bunch of other data types.

Reply:
:+1: